### ISSUE TITLE
confuses of predicted depth value

### ISSUE THREAD

**Issue by accuracy-maker** [CapsuleEndoscope/EndoSLAM#26]
Firstly, thanks for your great work. 

I downloaded the dataset and used 10 images of UnityCam folder. And I used the pre-trained dispnet to inference the depth. However, when I did some analysis of results, I found that the color maps of `ground truth grey` and `predicted_depth` are totally opposite. And I printed out the max values of two matrix.

![image](https://github.com/user-attachments/assets/978fb3f5-9cc5-468c-a802-9862c6087d2c)

The magnitude of the two values is different.

I am looking for some helps of this problem.

Thanks
